.gitignore

```diff
@@ -20,4 +20,5 @@ xcuserdata/
 ## Other
 *.moved-aside
 *.xccheckout
-*.xcscmblueprint
\ No newline at end of file
+*.xcscmblueprint
+.DS_Store
```

Comment:
Can you please make `*.DS_Store`? This should make it applicable to any `.DS_Store` in any child folders.

Reply:
Done!